src/energyplus/Test/HeatExchangerDesiccantBalancedFlowPerformanceDataType1_GTest.cpp

```diff
@@ -0,0 +1,277 @@
+                 false)
+      .get());
+  EXPECT_EQ(
+    48.88889,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumRegenerationInletAirTemperatureforHumidityRatioEquation,
+                 false)
+      .get());
+  EXPECT_EQ(
+    0.005000,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumProcessInletAirHumidityRatioforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    0.015714,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumProcessInletAirHumidityRatioforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    4.583333,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumProcessInletAirTemperatureforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    21.83333,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumProcessInletAirTemperatureforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    2.286,
+    idfHXP.getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumRegenerationAirVelocityforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    4.826,
+    idfHXP.getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumRegenerationAirVelocityforHumidityRatioEquation, false)
+      .get());
+  EXPECT_EQ(
+    0.007811,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumRegenerationOutletAirHumidityRatioforHumidityRatioEquation,
+                 false)
+      .get());
+  EXPECT_EQ(
+    0.026707,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumRegenerationOutletAirHumidityRatioforHumidityRatioEquation,
+                 false)
+      .get());
+  EXPECT_EQ(
+    10.0,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumRegenerationInletAirRelativeHumidityforHumidityRatioEquation,
+                 false)
+      .get());
+  EXPECT_EQ(
+    100.0,
+    idfHXP
+      .getDouble(HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumRegenerationInletAirRelativeHumidityforHumidityRatioEquation,
+                 false)
+      .get());
+  EXPECT_EQ(
+    80.0, idfHXP
+            .getDouble(
+              HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MinimumProcessInletAirRelativeHumidityforHumidityRatioEquation, false)
+            .get());
+  EXPECT_EQ(
+    100.0, idfHXP
+             .getDouble(
+               HeatExchanger_Desiccant_BalancedFlow_PerformanceDataType1Fields::MaximumProcessInletAirRelativeHumidityforHumidityRatioEquation, false)
+             .get());
+}
```

Comment:
I see no RT test. I assume this is because there isn't a RT?

Reply:
Correct, no RT. I didn't see RT for any of the other HeatExchanger objects, so figured I'd follow suit.